SharedPasswords/Credential.swift

```diff
@@ -7,6 +7,8 @@
 
 import Foundation
 
+public let sharedCredentialsKey = "sharedCredentials"
```

Comment:
I don't really like having this live at the top level. I'd rather have it live inside of something. Typically, I like to have a `Keys` struct that has `static let`s inside it with each of the needed keys. Maybe we could put that in the file with the events since that is kind of the file with shared content.